src/common/uriHelpers.ts

```diff
@@ -16,6 +17,7 @@ export function parseUriCommon(
   currencyInfo: EdgeCurrencyInfo,
   uri: string,
   networks: { [network: string]: boolean },
+  builtinTokens: EdgeTokenMap,
```

Comment:
Nitpick: In the `getLegacyDenomination`, you put the `builtinTokens` last, but here you put it in the middle. Maybe it should go at the end here too, to match what `getLegacyDenomination` does?